zoe-cli/test/com/adevinta/oss/zoe/cli/helpers.kt

```diff
@@ -72,17 +76,28 @@ suspend fun ExpectScope.zoe(
     return res
 }
 
-val testConfDir = File("testResources/env")
+private val testModule = module {
+    single { customizeEnvConfig }
```

Comment:
I think you could instead override the whole `EnvConfig` by an instance created from scratch for the purpose of testing (see [my above comment](https://github.com/adevinta/zoe/pull/6/files#r456514406))